src/Console/Command/SelfUpdateCommand.php

```diff
@@ -69,17 +70,24 @@ protected function execute(InputInterface $input, OutputInterface $output)
             return 1;
         }
 
-        $checker = new NewVersionChecker('v1.1.0');
+        $checker = new NewVersionChecker($this->getApplication()->getVersion());
+
+        try {
+            $latestVersion = $checker->getLatestVersion();
+            $latestVersionOfCurrentMajor = $checker->getLatestVersionOfCurrentMajor();
+        } catch (\Exception $exception) {
+            $output->writeln('<error>Unable to determine newest version.</error>');
```

Comment:
maybe print the exception message to the output for details as well (could be done based on verbosity level maybe)?

Reply:
Exception message added but not based on verbosity level: IMO you don't want to run the command again just to know why it failed. Also this would require you to be aware that this error message will be shown when increasing verbosity.

Reply:
makes total sense, thanks man :)